assets/css/style.css

```diff
@@ -68,6 +68,10 @@ span{
 }
 main{
     padding: 1rem;
+    background-image: url(../images/noo1\ \(5\).jpg);
```

Comment:
The image is not seen as u can see below, but in the inspect I can see an image was added in the main.
![Screenshot from 2021-09-15 06-19-27](https://user-images.githubusercontent.com/32802973/133365240-ebfe8235-4c95-45e8-b248-60624959c507.png)


In the inspect when I click to only see the image this is what I see 
![Screenshot from 2021-09-15 06-19-50](https://user-images.githubusercontent.com/32802973/133365342-dd5ed732-0f41-4713-9ce6-598a60a6b717.png)

when I check the repo, I see that u PUT SPACES IN THE IMAGE name. Bro!
![Screenshot from 2021-09-15 06-18-56](https://user-images.githubusercontent.com/32802973/133365198-d532e726-5ad8-482f-9b8e-4e28ba997855.png)

Reply:
done doing all the requested changes... and even done pushing back branch15